prog/waveplot/initwaveplot.F90

```diff
@@ -441,20 +441,19 @@ subroutine readOptions(node, identity, nLevel, nKPoint, nSpin, occupations, tRea
           &"Wave functions are real, no imaginary part will be plotted")
       tPlotImag = .false.
     end if
-    call getChildValue(node, "PlottedLevels", buffer, child=field, &
-        &multiple=.true.)
-    call convRangeToInt(char(buffer), node, plottedLevels, nLevel)
+    call getChildValue(node, "PlottedLevels", buffer, child=field, multiple=.true.)
+    call getSelectedIndices(node, char(buffer), [1, nLevel], plottedLevels)
     if (tPeriodic) then
       call getChildValue(node, "PlottedKPoints", buffer, child=field, &
           &multiple=.true.)
-      call convRangeToInt(char(buffer), node, plottedKPoints, nKPoint)
+      call getSelectedIndices(node, char(buffer), [1, nKPoint], plottedKPoints)
```

Comment:
k-points would be useful to have a stride operator (every 10th k-point for example).